cli/exec.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+	"sync"
+
+	"github.com/alibaba/pouch/apis/types"
+
+	"github.com/spf13/cobra"
+)
+
+// ExecCommand is used to implement 'exec' command.
+type ExecCommand struct {
+	baseCommand
+	Interactive bool
+	Terminal    bool
+	Detach      bool
+}
+
+// Init initializes ExecCommand command.
+func (e *ExecCommand) Init(c *Cli) {
+	e.cli = c
+	e.cmd = &cobra.Command{
+		Use:   "exec",
```

Comment:
Could you add more details about the command usage, like @Letty5411 mentioned in PR https://github.com/alibaba/pouch/issues/129, https://stackoverflow.com/questions/9725675/is-there-a-standard-format-for-command-line-shell-help-text
```
usage: app_name [options] required_input required_input2
  options:
    -a, --argument     Does something
    -b required     Does something with "required"
    -c, --command required     Something else
    -d [optlistitem1 optlistitem 2 ... ]     Something with list
```